stack/base/config-quick-setup.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 db_engine: postgresql_psycopg2
-db_backend: psql_dos
+db_backend: core.psql_dos
```

Comment:
Why did this work before? Or is this change to just squash a deprecation warning?

Reply:
It was not working before, this is the reason why simply bump the version to 2.1 fail the test at first place. This is a API change introduced in aiida-core 2.1 https://github.com/aiidateam/aiida-core/blob/main/CHANGELOG.md#pluginable-data-storage-backends

Reply:
I am very surprised they did not change the entry point in a backwards compatible manner, as with previous analogous changes. Is that actually a bug that should be reported? cc @sphuber 

(the change here is fine of course, thank you for the explanation)

Reply:
Hmm, this we might have missed. The change was intended to be backwards-compatible. Existing profiles with `psql_dos` *are* automatically migrated to `core.psql_dos`, but indeed, the option for `verdi quicksetup/setup` now no longer accepts `psql_dos`.

Reply:
@sphuber thanks for chiming in. Note that while migrating we also hit some other backwards incompatible changes when setting up the computer, I think transport `core.local` and scheduler, e.g. `core.slurm`, see changes in this PR.

https://github.com/ispg-group/aiidalab-ispg-docker-stack/pull/11/files#diff-602080df8884ed8d19826da3ccee7524f675094c9909ddcb3b9ebe8b6847696aR13

Reply:
Yeah, same problem. It is quite tricky to make these things backwards compatible. For option values we could potentially implement some deprecation pathway by temporarily allowing old values and in the command body detecting them and converting to the new entry point string. However, there are also subcommands that are based on the entry point string, for example `verdi computer configure local` now has become `verdi computer configure core.local` and this is very tricky to keep a deprecation pathway.

But in the end, all of this was added in v2.0 (https://github.com/aiidateam/aiida-core/commit/16daad08e63f0325ecd3fdaed7b2f7f8001ddaa7) so we considered these small breaking changes to be acceptable in a major version bump.